featuretools/tests/primitive_tests/test_transform_primitive.py

```diff
@@ -83,3 +83,14 @@ def test_age_nan():
     ages = age(dates, time=datetime(2020, 2, 26))
     correct_ages = [10.159, np.nan, 8.159]
     np.testing.assert_array_almost_equal(ages, correct_ages, decimal=3)
+
+
+def test_no_deprecation_message():
```

Comment:
let's call this "test_week_no_deprecation_message"

Reply:
:+1: